packages/documentation/src/stories/components/custom-select/custom-select.docs.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Source } from '@storybook/blocks';
+import * as CustomSelectStories from './custom-select.stories.ts';
+import StylesPackageImport from '../../../shared/styles-package-import.mdx';
+import { PostAlert, PostTabs, PostTabHeader, PostTabPanel } from '@swisspost/design-system-components-react';
+import SampleBasicHTML from './custom-select-basic.sample.html?raw';
+import SampleBasicTS from './custom-select-basic.sample.ts?raw';
+import SampleFloatingHTML from './custom-select-floating.sample.html?raw';
+import SampleFloatingTS from './custom-select-floating.sample.ts?raw';
+
+<Meta of={CustomSelectStories} />
+
+<div className="d-flex align-items-center justify-content-between">
+  # Custom-Select
+
+  <a
+  className="btn btn-rg btn-secondary btn-animated"
+  href={`https://archive.design-system.post.ch/#/post-samples/custom-select`}
+  target="_blank"
+  rel="nofollow noopener noreferrer"
+>
+  <span>See archived documentation</span>
+</a>
+</div>
+
+<p className="lead">A custom select for NgBootstrap that uses the ngbDropdown.</p>
+
+<PostAlert type="warning">
+<h4 slot="heading">This component is deprecated</h4>
+<p>It will be removed in the next major version.</p>
+</PostAlert>
+
+<ul>
+  <li>
+    <a href="#component-import" target="_self">Component Import</a>
+  </li>
+  <li>
+    <a href="#style-imports" target="_self">Style Imports</a>
+  </li>
+  <li>
+    <a href="#examples" target="_self">Examples</a>
+  </li>
+</ul>
+
+## Component Import
+
+This component uses the [Dropdown](/?path=/docs/components-dropdown--docs) component that we also documented. There is no additional NgBootstrap import required.
+
+<StylesPackageImport components={["forms", "form-select"]}
+                     required={{ 'floating-label': true }} />
+
+## Examples
+
+<PostTabs>
+  <PostTabHeader slot="tabs" panel="sampleBasic">Basic example</PostTabHeader>
+  <PostTabPanel name="sampleBasic">
+    <Source code={SampleBasicHTML} language="html" />
+    <Source code={SampleBasicTS} language="typescript"/>
```

Comment:
If you think about accessibility, it would be great to have headings before each file.